kolibri/core/mixins.py

```diff
@@ -28,22 +30,33 @@ class BulkDeleteMixin(object):
 
     # Taken from https://github.com/miki725/django-rest-framework-bulk
 
-    def allow_bulk_destroy(self, qs, filtered):
+    def allow_bulk_destroy(self, qs):
         """
         Hook to ensure that the bulk destroy should be allowed.
         By default this checks that the destroy is only applied to
         filtered querysets.
         """
+        filter_fields = set()
```

Comment:
This looks like the right change to me - sufficiently general to catch the use cases we need, but not going overboard!